fabtests/pytest/efa/test_rdm.py

```diff
@@ -78,3 +78,12 @@ def test_rdm_tagged_peek(cmdline_args):
     test = ClientServerTest(cmdline_args, "fi_rdm_tagged_peek", timeout=1800)
     test.run()
 
+# This test is run in serial mode because it takes a lot of memory
+@pytest.mark.serial
+@pytest.mark.functional
+def test_rdm_pingpong_1G(cmdline_args, completion_semantic, memory_type):
+    # Default window size is 64 resulting in 128GB being registered, which
+    # exceeds max number of registered host pages
+    efa_run_client_server_test(cmdline_args, "fi_rdm_pingpong -W 1", "short",
```

Comment:
I don't think ping-pong uses windows it's just send/recv in sequence, it's BW test like `rdm_tagged_bw` uses window.

Reply:
I see, windows size is used when registering the big buffer at the beginning.